core/toolbox.js

```diff
@@ -470,23 +533,23 @@ Blockly.Toolbox.prototype.updateColourFromTheme = function() {
 
 /**
  * Updates the background colour of the selected category.
- * @param {!Blockly.Toolbox.TreeNode} tree Starting point of tree.
+ * @param {!Blockly.tree.TreeNode} tree Starting point of tree.
  *     Defaults to the root node.
  * @private
  */
 Blockly.Toolbox.prototype.updateSelectedItemColour_ = function(tree) {
-  var selectedItem = tree.selectedItem_;
+  var selectedItem = tree.getSelectedItem();
   if (selectedItem) {
     var hexColour = selectedItem.hexColour || '#57e';
     selectedItem.getRowElement().style.backgroundColor = hexColour;
-    tree.toolbox_.addColour_(selectedItem);
+    tree.getToolbox().addColour_(selectedItem);
```

Comment:
Is `tree.getToolbox()` the same as `this`?

Reply:
Yeah haha I wonder why we were doing tree.toolbox_